js/main.js

```diff
@@ -27,4 +27,96 @@ function handleSubmit(event) {
   data.entries.unshift(inputValues);
   $image.setAttribute('src', 'images/placeholder-image-square.jpg');
   $entryForm.reset();
+  renderEntry(0);
```

Comment:
You shouldn't be passing in a number, you should be passing in your `inputValues` function